[GLOBAL]
print_exception_stacktrace: True

[python-repos]
